shared/src/util/markdown.test.ts

```diff
@@ -1,6 +1,73 @@
 import { renderMarkdown } from './markdown'
+import { registerHighlightContributions } from '../highlight/contributions'
+
+registerHighlightContributions()
 
 describe('renderMarkdown', () => {
+    it('renders code blocks, with syntax highlighting', () => {
+        const markdown = [
+            '# This is a heading',
+            '',
+            '## This is a subheading',
+            '',
+            'Some text',
+            'in the same paragraph',
+            'with a [link](./destination).',
+            '',
+            '```ts',
+            'const someTypeScriptCode = funcCall()',
+            '```',
+            '',
+            '- bullet list item 1',
+            '- bullet list item 2',
+            '',
+            '1. item 1',
+            '1. item 2',
```

Comment:
One edge-case that didn't work in our docs pages,: 

```suggestion
            '1. item 1',
            '  ```ts'
            '  const a = "string"',
            '  ```'
            '1. item 2',
```